third_party/tutorial.proto

```diff
@@ -45,13 +45,10 @@ message BlockContent {
     Image image = 2;
     InfoBox info_box = 3;
     CodeBlock code_block = 4;
-    HorizontalRule horizontal_rule = 5;
     Table table = 6;
-  }
-};
 
-message HorizontalRule {
-  // No fields.
+    reserved 5, "horizontal_rule";
```

Comment:
I think it's not possible to combine numbers and names. Try:

reserved 5;
reserved "horizontal_rule";

Reply:
You are correct.